src/lib/style/components/_global-alert.scss

```diff
@@ -26,6 +26,9 @@
     .messageArea {
       max-width: 700px;
       grid-row: 1 / span 1;
+      p {
+        margin-bottom: 0px;
+      }
```

Comment:
`Typography` uses `<p>` elements, and bootstrap 4 has the following rule (or something like it):
```css
.bootstrap-4-backport {
	p {
		margin-bottom: 1rem;
	}
}
``` 

So this will override it.